inference-engine/src/transformations/src/transformations/convert_precision.cpp

```diff
@@ -0,0 +1,211 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/convert_precision.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/opsets/opset3.hpp>
+#include <ngraph/opsets/opset1.hpp>
+
+using namespace ngraph;
+
+bool fuse_type_to_constant(std::shared_ptr<ngraph::Node> node, ngraph::element::Type to, const std::vector<ngraph::Input<ngraph::Node>> & consumers);
```

Comment:
Do you change `node` in these methods?
You can add `&` in order to don't call copy ctor for shared_ptr.

Reply:
done